commit 2618e4421663f0f6edb6598e417b9be47a868ac5
Author: Sergio Correia <sergio@correia.cc>
Date:   Mon Sep 5 20:16:53 2016 -0400

    Revert "Use lowercase name for package folders (#2162)"
    
    This reverts commit c695d41c47c8baa48db1a590fe7378641a9e0ab9.

diff --git a/src/TypeCatalogParser/Main.cs b/src/TypeCatalogParser/Main.cs
index 91ec7e9..3e7f07b 100644
--- a/src/TypeCatalogParser/Main.cs
+++ b/src/TypeCatalogParser/Main.cs
@@ -34,7 +34,7 @@ namespace TypeCatalogParser
                                          // Get the real reference assemblies
                                          from y in x.CompileTimeAssemblies where y.Path.EndsWith(".dll")
                                          // Construct the path to the assemblies
-                                         select $"{context.PackagesDirectory}/{x.Name.ToLower()}/{x.Version}/{y.Path};");
+                                         select $"{context.PackagesDirectory}/{x.Name}/{x.Version}/{y.Path};");
 
             Console.WriteLine($"List of reference assemblies written to {outputPath}");
         }
